docs/lib/predictions/fragments/ios/identify-text.md

```diff
@@ -50,6 +50,25 @@ func detectText(_ image: URL) {
 }
 ```
 
+**Note**: Bounding boxes in IdentifyTextResult are returned as ratios. If you would like to place bounding boxes on individual recognized words that appear in the image, use the following method to calculate a frame for a single bounding box.
+```swift 
+	let image = UIImage(...)
+	self.imageView = UIImageView(image: self.image)                        
+	...
+	let data = result as! IdentifyTextResult
+	let word = data.words!.first!
+	...
+	let xBoundingBox = word.boundingBox.origin.x * self.imageView.frame.size.width
+	let yBoundingBox = word.boundingBox.origin.y * self.imageView.frame.size.height
+	let widthBoundingBox = word.boundingBox.size.width * self.imageView.frame.size.width
+	let heightBoundingBox = word.boundingBox.size.height * self.imageView.frame.size.height
+	let boundingBox = UIView(frame: CGRect(x: xBoundingBox, y: yBoundingBox, width: widthBoundingBox, height: heightBoundingBox))
+	... // stylize bounding box
+	self.imageView.addSubview(boundingBox)
```

Comment:
Ok, so this code as pointed out by @brene is difficult to copy and paste.  So lets do this
1.  Update the code above to have a completion block like the following:
```
    func detectText(_ image: URL, completion: @escaping ([IdentifiedWord]) -> Void) {
        _ = Amplify.Predictions.identify(type: .detectText(.plain), image: image, options: PredictionsIdentifyRequest.Options(), listener: { (event) in
            switch event {
            case .completed(let result):
                let data = result as! IdentifyTextResult
                completion(data.words!)
            case .failed(let error):
                print(error)
            default:
                print("")
            }
        })
    }
```
2. Update the bounding box example to be something like this:
```
    @IBAction func didTapButton(_ sender: Any) {
        let imageURL = URL(string: "https://imageWithText")
        let data = try? Data(contentsOf: imageURL!)
        let image = UIImage(data: data!)
        let imageView = UIImageView(image: image)
        self.view.addSubview(imageView)

        detectText(imageURL!, completion: { words in
            let word = words.first!
            DispatchQueue.main.async {
                let xBoundingBox = word.boundingBox.origin.x * imageView.frame.size.width
                let yBoundingBox = word.boundingBox.origin.y * imageView.frame.size.height
                let widthBoundingBox = word.boundingBox.size.width * imageView.frame.size.width
                let heightBoundingBox = word.boundingBox.size.height * imageView.frame.size.height
                let boundingBox = UIView(frame: CGRect(x: xBoundingBox, y: yBoundingBox, width: widthBoundingBox, height: heightBoundingBox))
                boundingBox.backgroundColor = .red
                imageView.addSubview(boundingBox)
            }
        })
    }
```

Reply:
updated